lib/diagnostics_channel.js

```diff
@@ -60,12 +131,30 @@ class ActiveChannel {
       }
     }
   }
+
+  runStores(data, fn, thisArg, ...args) {
```

Comment:
The name `runStores` doesn't really make it clear that it also publishes, although I don't have any better suggestion, just pointing out that I never remember what this does from the name alone every time I look at this PR.

Reply:
`runStoresAndPublish`? Sound a bit Java-ish. 😅

Reply:
I don't really have an alternative in mind, just saying that `runStores` doesn't really mean anything.